src/modules/simulator/simulator_mavlink.cpp

```diff
@@ -162,6 +162,17 @@ mavlink_hil_actuator_controls_t Simulator::actuator_controls_from_outputs()
 			}
 		}
 
+	} else if (_system_type == MAV_TYPE_SUBMARINE) {
+		/* scale all outputs to -1..1 for UUVs */
+		for (unsigned i = 0; i < 16; i++) {
+			if (armed) {
+				msg.controls[i] = (_actuator_outputs.output[i] - pwm_center) / ((PWM_DEFAULT_MAX - PWM_DEFAULT_MIN) / 2);
+
+			} else {
+				msg.controls[i] = 0.0f;
+			}
+		}
+
```

Comment:
I think this can be handled in the switch case above, and set `n=0`

Reply:
Of course! Greatly appreciate the advice

Reply:
Said and done! Thanks for the hint! 7a8bc04